### ISSUE TITLE
Use button semantics and styles for link elements that work as buttons

### ISSUE THREAD

**Issue by JasonYeMSFT** [microsoft/AzureStorageExplorer#7700]
Standard to meet:
- Elements that trigger "in-application actions" should be `<button>` elements. Accessibility Insights should register the element as a "button".
- Elements that "open link in external browsers" should be `<a>` elements, with a blue underline. Accessibility Insights should register the element as a "link".

```[tasklist]
## Elements needing updating
- [x] Action buttons of the renderer window notification (customized actions and the default close action)
```

**Comment by MRayermannMSFT** [comment]
In 1.34, please fill out the "Elements needing updating:" list and then we can determine how we want to triage/distribute the actual doing of this work.

**Comment by JasonYeMSFT** [comment]
Investigation is done for 1.34.0.
